app/infrastructure/database/repositories.py

```
@@ -0,0 +1,265 @@
+        except ContextNotFoundError:
+            raise
+        except Exception as e:
+            logger.error(f"Error updating context status: {str(e)}")
+            raise DatabaseError(f"Failed to update context: {str(e)}")
+
+
+class ContextCodeChunkInterface(ABC):
+    @abstractmethod
+    async def store_emebeddings(
+        self, repo_id: str, user_id: str, data: dict, commit_number: str
+    ) -> Optional[CodeChunks]:
```

Ruff (0.11.9)

`CodeChunks` may be undefined, or defined from star imports

(F405)